pantheon-bundle/frontend/src/app/versions.test.tsx

```diff
@@ -304,39 +283,39 @@ describe('Versions tests', () => {
-        })
-        const spy = sinon.spy(instance, 'changePublishState')
-        wrapper.find(Button).at(2).simulate('click')
-        sinon.assert.called(spy)
-    })
-
-    it('test fetchProducts function', () => {
-        const wrapper = renderer.create(<Versions {...props} />)
-        const inst = wrapper.getInstance()
-        const spy = sinon.spy(inst, 'fetchProducts')
-        inst.componentDidMount()
-        sinon.assert.called(spy)
-    })
-
-    it('test fetchVersions function', () => {
-        const wrapper = renderer.create(<Versions {...props} />)
-        const inst = wrapper.getInstance()
-        const spy = sinon.spy(inst, 'fetchVersions')
-        inst.fetchVersions()
-        sinon.assert.called(spy)
-    })
-
-    it('test changePublishState function', () => {
-        const wrapper = renderer.create(<Versions {...props} />)
-        const inst = wrapper.getInstance()
-        const spy = sinon.spy(inst, 'changePublishState')
-        inst.changePublishState("ds")
-        sinon.assert.called(spy)
-    })
+    // test('changePublishState click', () => {
+    //     const wrapper = mount(<Versions {...props} />)
+    //     const instance = wrapper.instance()
+    //     wrapper.setState({ 'login': true })
+    //     wrapper.setState({
+    //         'results': [[{ "type": "draft", "icon": "BlankImage", "path": "/modules/test", "version": "Version 1", "publishedState": 'Not published', "updatedDate": "", "firstButtonType": 'primary', "secondButtonType": 'secondary', "firstButtonText": 'Publish', "secondButtonText": 'Preview', "isDropdownOpen": false, "isArchiveDropDownOpen": false, "metadata": '' }]],
+    //     })
+    //     const spy = sinon.spy(instance, 'changePublishState')
+    //     wrapper.find(Button).at(2).simulate('click')
+    //     sinon.assert.called(spy)
+    // })
+
+    // it('test fetchProducts function', () => {
+    //     const wrapper = renderer.create(<Versions {...props} />)
+    //     const inst = wrapper.getInstance()
+    //     const spy = sinon.spy(inst, 'fetchProducts')
+    //     inst.componentDidMount()
+    //     sinon.assert.called(spy)
+    // })
+
+    // it('test fetchVersions function', () => {
+    //     const wrapper = renderer.create(<Versions {...props} />)
+    //     const inst = wrapper.getInstance()
+    //     const spy = sinon.spy(inst, 'fetchVersions')
+    //     inst.fetchVersions()
+    //     sinon.assert.called(spy)
+    // })
+
+    // it('test changePublishState function', () => {
+    //     const wrapper = renderer.create(<Versions {...props} />)
+    //     const inst = wrapper.getInstance()
+    //     const spy = sinon.spy(inst, 'changePublishState')
+    //     inst.changePublishState("ds")
+    //     sinon.assert.called(spy)
+    // })
```

Comment:
It looks like you've commented out this entire file. Was that intentional? Will the commented code be coming back at some point?

IIRC, versions.tsx is responsible for displaying most of the document details pages. I would guess that you've reworked most of that and that the old tests are now obsolete. If that's true, please remove them rather than commenting them.

Reply:
versions.test.tsx has been updated.